src/Components/EncodingString/EcodingString.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import "./EncodingStringS.scss";
+import { StringInputForm } from "./StringInputForm";
+import { encode } from "./utils";
+
+export const EncodingString = () => {
+    const [string, setstring] = useState<string>("");
+    const [encodeString, setEncodeString] = useState<string>(encode(string));
```

Comment:
string is not a great name, should use more descriptive names even in non production code.